.github/workflows/CI.yml

```diff
@@ -0,0 +1,33 @@
+---
+name: CI
+'on':
+  push:
+  workflow_dispatch:
+
+jobs:
+   tests:
+    runs-on: ubuntu-latest
+    timeout-minutes: 10
+    steps:
+      - uses: actions/checkout@v3
+      - name: build docker-compose services
+        run: docker-compose build
+      - name: test
+        run: |
+          docker-compose run \
+            -e DATABASE_URL="sqlite:///db3.sqlite3" \
+            -e DJANGO_SECRET_KEY="test" \
+            -e TEST_OUTPUT_FILE_NAME="junit.xml" \
+            -v "${{ github.workspace }}:/code" \
+            --no-deps \
+            --entrypoint "python manage.py test dice" \
```

Comment:
to avoid overriding things in CLI let's use additional docker-compose file for CI builds ant define there default `cmd` (because `cmd` should be used instead of `entrypoint` to run tests). 

References:

+ https://stackoverflow.com/questions/21553353/what-is-the-difference-between-cmd-and-entrypoint-in-a-dockerfile